frontend/test/__support__/cypress.js

```diff
@@ -81,6 +81,10 @@ export function openProductsTable() {
   cy.visit("/question/new?database=1&table=1");
 }
 
+export function openPeopleTable() {
+  cy.visit("/question/new?database=1&table=3");
+}
+
```

Comment:
Maybe these could be `openTable("table name")` instead? They should also probably use `withSampleData`

Reply:
I think those changes should be in a different PR because `openProductsTable()` and `openOrdersTable()` are dependeded on for a lot of different files.